### DISCUSSION TITLE
2.1.8 When?

### DISCUSSION THREAD

**Discussion by AlexUrrutia** [pedroSG94/RootEncoder#1095] [Q&A] [answered]
Hello Pedro, any idea when the 2.1.8 will be released?

Thanks!

**Comment by pedroSG94** [comment]
I have to finish 2 branches and then I can release 2.1.8 but I'm not sure about a date

**Reply by AlexUrrutia** [reply]
How can I compile the latest commit? Any link? Just want to test some functions with the latest updates, thank you!

**Comment by pedroSG94** [comment]
You can use this for now:
```gradle
  implementation 'com.github.pedroSG94.rtmp-rtsp-stream-client-java:rtplibrary:b98e8d9647'
```

**Reply by AlexUrrutia** [reply]
Thanks!

**Comment by AlexUrrutia** [comment]
Hello Pedro, can I get the latest commit path to test the latest fixes? thanks! @pedroSG94

**Comment by pedroSG94** [comment, marked as answer]
Hello,

The current version was 2.1.9. Anyway I update the library to the last commit using version 2.2.0
